packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,321 @@
+            setHideLabelsLayout(false);
+          } else if (layout === StepperLayout.summaryView) {
+            setSummaryViewLayout(false);
+          }
+          if (breakpointLayout === StepperLayout.showLabels) {
+            setShowLabelsLayout(true);
+          } else if (breakpointLayout === StepperLayout.hideLabels) {
+            setHideLabelsLayout(true);
+          } else if (breakpointLayout === StepperLayout.summaryView) {
+            setSummaryViewLayout(true);
+          }
+        }
+        if (window.innerWidth > breakpoint || !breakpoint) {
+          if (breakpointLayout === StepperLayout.showLabels) {
+            setShowLabelsLayout(false);
+          } else if (breakpointLayout === StepperLayout.hideLabels) {
+            setHideLabelsLayout(false);
+          } else if (breakpointLayout === StepperLayout.summaryView) {
+            setSummaryViewLayout(false);
+          }
+          if (layout === StepperLayout.showLabels) {
+            setShowLabelsLayout(true);
+          } else if (layout === StepperLayout.hideLabels) {
+            setHideLabelsLayout(true);
+          } else if (layout === StepperLayout.summaryView) {
+            setSummaryViewLayout(true);
+          }
+        }
+        if (showLabelsLayout) {
+          setHideLabelsLayout(false);
+          setSummaryViewLayout(false);
+        } else if (hideLabelsLayout) {
+          setShowLabelsLayout(false);
+          setSummaryViewLayout(false);
+        } else if (summaryViewLayout) {
+          setShowLabelsLayout(false);
+          setHideLabelsLayout(false);
+        }
+      }
+
+      window.addEventListener('resize', handleResize);
+      handleResize();
+
+      return () => window.removeEventListener('resize', handleResize);
+    }, [windowWidth]);
+
+    // Step states
+    const step = React.Children.toArray(children);
+
+    const Steps = React.Children.map(
+      children,
+      (
+        child: React.ReactElement<React.PropsWithChildren<StepProps>>,
+        index
+      ) => {
+        if (child.type === Step) {
+          const stepStatusStyles =
+            currentStep >= index + 1
+              ? StepStatus.complete
+              : currentStep >= index
+              ? StepStatus.active
+              : StepStatus.incomplete;
+
+          const item = React.cloneElement(child, {
+            'aria-current': currentStep === index,
+            children: currentStep === index ? child : null,
+            key: index,
+            isInverse: isInverse,
+            hasHideLabels: hideLabelsLayout,
+            hasSummaryView: summaryViewLayout,
+            stepStatus: stepStatusStyles,
+          });
+
+          return (
+            <>
+              {item}
+              {index !== step.length - 1 && (
+                <StyledSeparator
+                  isInverse={isInverse}
+                  stepStatus={stepStatusStyles}
+                  theme={theme}
+                />
+              )}
+            </>
```

Comment:
I'm debating about this, and would love your thoughts. 

Currently, for the last child it returns the child and then false. 
![image](https://github.com/cengage/react-magma/assets/91160746/46a27033-22ec-4c78-ac89-59b370c41e0b)

We could use this array notation to prevent the unnecessary false child
```
[
  item,
  index !== step.length - 1 && (
    <StyledSeparator
      key={`separator-${index}`}
      isInverse={isInverse}
      stepStatus={stepStatusStyles}
      theme={theme}
    />
  )
]
```
![image](https://github.com/cengage/react-magma/assets/91160746/501a4c0d-e26e-4edf-b364-5877492cc144)